diff -Nru libdbi-0.6.7.orig/configure.in libdbi-0.6.7/configure.in
--- libdbi-0.6.7.orig/configure.in	2002-11-18 06:54:16.000000000 +0000
+++ libdbi-0.6.7/configure.in	2003-06-08 22:53:37.000000000 +0000
@@ -83,7 +83,6 @@
 dnl ==============================
 
 AC_CHECK_MYSQL
-AC_CHECK_PGSQL
 
 dnl ==============================
 dnl Check for functions
@@ -117,6 +116,5 @@
 	include/dbi/Makefile
 	drivers/Makefile
 	drivers/mysql/Makefile
-	drivers/postgresql/Makefile
 	tests/Makefile
 	])
diff -Nru libdbi-0.6.7.orig/drivers/Makefile.am libdbi-0.6.7/drivers/Makefile.am
--- libdbi-0.6.7.orig/drivers/Makefile.am	2001-07-19 08:15:25.000000000 +0000
+++ libdbi-0.6.7/drivers/Makefile.am	2003-06-08 22:52:36.000000000 +0000
@@ -1,2 +1,2 @@
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = mysql postgresql
+SUBDIRS = mysql
