--- libdbi/doc/programmers-guide.sgml~	2008-03-04 10:00:35.000000000 +0100
+++ libdbi/doc/programmers-guide.sgml	2009-04-20 14:53:38.326280343 +0200
@@ -122,9 +122,9 @@
     &libdbi-errorhandling;
   </Chapter>
 
-  <chapter id="versioning"><title>Library and Interface Versions</Title>
+  <!--  <chapter id="versioning"><title>Library and Interface Versions</Title>
     &libdbi-versioning;
-  </Chapter>
+</Chapter>-->
 
   &libdbi-reference;
   &freedoc-license;
